tests/manager/manager-ctor-tls-error-003.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): tlsInsecure cannot be combined with tlsDisableOCSPEndpointCheck
```

Comment:
I assume these tests pertain to [How OCSP interacts with existing configuration options](https://github.com/mongodb/specifications/blob/master/source/ocsp-support/ocsp-support.rst#how-ocsp-interacts-with-existing-configuration-options). The URI spec tests cover all four permutations (e.g. [4 permutations of tlsInsecure=true/false, tlsDisableOCSPEndpointCheck=true/false]( https://github.com/mongodb/specifications/blob/433e6a93e37de8764c18d6bff1be598eb4c26586/source/uri-options/tests/tls-options.yml#L447).

I assume we don't want to test each permutation, as it would be far more than four for every combination of URI string and array option. That said, would it make sense to mix these tests up a bit and use `false` for some cases?

Reply:
Good idea. I consolidated all 6 test files into a single test. For all 6 invalid combinations we're now testing true/false for each option. However, instead of also testing 4 permutations of combining the options through URI and the options array, we're now testing passing both of them in the URI, and passing one option in the URI and one in the options array. These tests should cover all significant permutations.

Let me know if you'd rather keep the different combinations in separate tests to have a more concise expected output, but given how these checks are all related I thought it was best to combine them in a single test.